modules/navigation.js

```diff
@@ -0,0 +1,24 @@
+const booksSection = document.querySelector('.books-section');
+const addSection = document.querySelector('.add-books');
+const contactSection = document.querySelector('.contact-section');
+
+export const displayBooks = (e) => {
+  e.preventDefault();
+  addSection.classList.remove('toggle');
+  booksSection.classList.add('toggle');
+  contactSection.classList.add('toggle');
+};
+
+export const displayAddForm = (e) => {
+  e.preventDefault();
+  booksSection.classList.remove('toggle');
+  addSection.classList.add('toggle');
+  contactSection.classList.add('toggle');
+};
+
+export const displayContact = (e) => {
+  e.preventDefault();
+  contactSection.classList.remove('toggle');
+  booksSection.classList.add('toggle');
+  addSection.classList.add('toggle');
+};
```

Comment:
- [ ] Kindly look into all these functions. Nothing seems to work here. Do you have any class name `toggle` or you really meant something else. At this moment, nothing navigate with click event. 

### App Behaviour
[app.webm](https://github.com/abdullah97-pro/Awesome-books-ES6/assets/26597432/e7e2e17a-f578-4648-8606-53c93583f25c)

Reply:
it is working for my correctly

Reply:
> * [x]  Kindly look into all these functions. Nothing seems to work here. Do you have any class name `toggle` or you really meant something else. At this moment, nothing navigate with click event.
> 
> ### App Behaviour
>  app.webm

it is working for me you please check again

Reply:
you have to run as a liveserver